src/imputation/imputation_main.py

```diff
@@ -35,7 +35,7 @@ def run_imputation(
 
     df = run_apportionment(df)
 
-    df = run_short_to_long(df)
+    df = run_short_to_long(df, selectiontype=["C"])
```

Comment:
This actually now needs to be selectiontype=["C", "P"]

Reply:
Changed